packages/node/src/types.ts

```diff
@@ -67,22 +67,18 @@ export interface ApiCall {
   readonly fulfillAddress: string | null;
   readonly fulfillFunctionId: string | null;
   readonly parameters: ApiCallParameters;
-  readonly providerId: string;
+  readonly providerId: string | null;
```

Comment:
Here, `providerId` should be treated the same as `endpointId` (full requests get it from request parameters, regular requests get it from the template)

Reply:
Good point 👍🏻